tests/test_pauli.py

```diff
@@ -1,25 +1,21 @@
 from __future__ import annotations
 
 import itertools
-from typing import TYPE_CHECKING
 
 import numpy as np
 import pytest
 
-import graphix.pauli
 from graphix.clifford import Clifford
 from graphix.command import MeasureUpdate
-
-if TYPE_CHECKING:
-    from graphix.pauli import ComplexUnit, Pauli, Plane
+from graphix.pauli import LIST, UNITS, ComplexUnit, Pauli, Plane
 
 
 class TestPauli:
     @pytest.mark.parametrize(
         ("u", "p"),
         itertools.product(
-            graphix.pauli.UNITS,
-            graphix.pauli.LIST,
+            UNITS,
+            LIST,
```

Comment:
I would prefer to use at least `pauli.LIST` while waiting for the refactoring of the Pauli namespace.

Reply:
I will soon (literally soon!) submit another PR for this.
Anyway let me use your idea here.

Reply:
Fixed in 7cf2e8383b0d905dd20b3cf0e275bf7523ed8c4c .